templates/partials/monster-inventory-tab.html

```diff
@@ -18,42 +18,50 @@ <h2>{{localize "WITCHER.Loot.Weapons"}} <a class="add-item" data-itemType="weapo
                 <img src="{{weapon.img}}" class="item-img dragable" draggable="true" data-id="{{weapon._id}}"/>
             </td>
             <td><a class="item-roll">{{weapon.name}}</a></td>
-            <td><input class="inline-edit" data-field="system.damage" type="text" value="{{weapon.system.damage}}" placeholder="" /></td>
-            <td><input class="inline-edit small" data-field="system.reliable" type="text" value="{{weapon.system.reliable}}" placeholder="" /></td>
-            <td><input class="inline-edit small" data-field="system.rateOfFire" type="text" value="{{weapon.system.rateOfFire}}" data-dtype="Number"/></td>
+            <td><input class="inline-edit" data-field="system.damage" type="text" value="{{weapon.system.damage}}"
+                       placeholder=""/></td>
+            <td><input class="inline-edit small" data-field="system.reliable" type="text"
+                       value="{{weapon.system.reliable}}" placeholder=""/></td>
+            <td><input class="inline-edit small" data-field="system.rateOfFire" type="text"
+                       value="{{weapon.system.rateOfFire}}" data-dtype="Number"/></td>
         </tr>
-        <tr class="item-info invisible" >
+        <tr class="item-info invisible">
             <td>
                 <a class="item-edit"><i class="fas fa-edit"></i></a>
             </td>
-            <td colspan="3">
-                <div class="monster-flex"><b>{{localize "WITCHER.Spell.Effect"}}:</b></label>
+            <td>
+                <div class="monster-flex">
+                    <label>{{localize "WITCHER.Spell.Effect"}}:</label>
+                </div>
+            </td>
+            <td colspan="2">
+                <div class="monster-flex">
                     {{#each weapon.system.effects as |effect id|}}
-                        <div class="monster-flex">
-                        {{effect.name}}
-                        {{#if effect.percentage}}
-                            ({{effect.percentage}})%   
-                        {{/if}}
-                        ,
-                        </div>
+                        <div class="monster-flex-small">{{effect.name}}
+                            {{#if effect.percentage}}
+                                ({{effect.percentage}})%
+                            {{/if}}</div>
```

Comment:
keep the coma so that the effects are separated by a coma

Reply:
IMO coma is redundant here
![image](https://user-images.githubusercontent.com/52590783/192093009-f5457336-82a7-461f-8ba3-857a2783ef83.png)

Actual list of elements is divided with new lines with help of monster-flex-small class

Reply:
image w/o coma 
![image](https://user-images.githubusercontent.com/52590783/192093091-705d43a6-8e6f-495c-a56d-482fa9c2151d.png)

Reply:
Ah thats good then, I tought they where all in one line.  Thank for the screen shots!